event.go

```diff
@@ -67,10 +70,22 @@ func (ed *DefaultEventHandler) Post(height uint64, msg *types.EventData) error {
 	}
 	// TODO: this is stupid, fix it
 	eventData := EventData(*msg)
-	ed.stash.add(height, &eventData)
+	ed.eventCache = append(ed.eventCache, &eventData)
 	return nil
 }
 
+func (ed *DefaultEventHandler) Commit(height uint64) error {
```

Comment:
Since it never returns an error might as well simplify the function signature.

Reply:
Done